main.bal

```diff
@@ -11,6 +12,122 @@ service on new tcp:Listener(3000) {
     }
 }
 
+    type Message record {
+        string jsonrpc;
+    };
+
+    type RequestMessage record {|
+        *Message;
+        int|string id;
+        string method;
+        any params?;
```

Comment:
use `anydata` instead `any`
(ideally this should be `json` but we can't do it due to a bug in current compiler.)